src/utils/sourcemaps.rs

```diff
@@ -992,6 +992,10 @@ impl SourceMapProcessor {
                             }
                         };
 
+                        // FIXME: This seems to be a false positive; clippy
+                        // thinks we're doing `contains_key` followed by `insert`,
+                        // but there's no `insert`.
+                        #[allow(clippy::map_entry)]
```

Comment:
Let's use an `expect`:

```suggestion
                        #[expect(clippy::map_entry)]
```

Seems that there is already a [PR which fixes this](https://github.com/rust-lang/rust-clippy/issues/14449), so with the expect, we'll be alerted to remove the `expect` once the PR is released

Reply:
Good find, I totally missed that, I guess because I was only looking at open issues.

Reply:
@loewenheim that is kinda sus, where are we linting with nightly?